pages/dkp/kommander/1.3/install-airgapped/index.md

```diff
@@ -4,7 +4,7 @@ navigationTitle: Install air gapped
 title: Install Kommander air gapped
 menuWeight: 35
 excerpt: Install Kommander in an air gapped environment
-beta: false
+beta: true
```

Comment:
Is this really beta?

Reply:
according to @drewvanstone it is.

Reply:
Yeah, just checked with Sam B. It's still beta.